scripts/tools-versions-minimum.txt

```diff
@@ -1,12 +1,9 @@
-python=3.6
-cmake=3.20.0
-ninja=1.8.2
+python=3.8.2
+cmake=3.20.5
```

Comment:
in Zephr SDK https://docs.zephyrproject.org/latest/develop/getting_started/index.html minimum version is 3.20.0